CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v1.7.0
+
+- Install flutter sdk in publishing step, allowing Flutter packages to be
+  published ([#68][]) 
```

Comment:
```suggestion
  published ([#68][68]) 
```

Reply:
Or you just write:
```md
published ([#68]) 

[#68]: https://github.com/dart-lang/setup-dart/issues/68
```

published ([#68]) 

[#68]: https://github.com/dart-lang/setup-dart/issues/68

Reply:
Not sure why anyone would explicitly make the empty `[]` at the end.

----

Also I don't think `[#64]` will match the `[64]: <url>` thing. I think it only works on github because github does automatic links for `#64`, but I could be wrong.